codespell_lib/data/dictionary.txt

```diff
@@ -10804,6 +10804,10 @@ dorder->order, disorder,
 dordered->ordered
 dorment->dormant
 dorp->drop
+dosclosed->disclosed
+doscloses->discloses
+dosclosing->disclosing
+dosclosure->disclosure
```

Comment:
May as well add this too:
```suggestion
dosclosure->disclosure
dosclosures->disclosures
```

Reply:
Yes, indeed. Thanks a lot for the suggestion :+1: